vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,166 @@
+
+4. To navigate to your Portal: \
+from the left menu →  `Page` → `Mangement` \
+find `Customer Portal` under `Page Tree` list \
+from the `Actions` pull-down menu →  `View Published Page`
+
+## Configuration
+### Administrators
+In order to configure the Customer Portal and its users, the administrator will need Jira Service Management and Entando Identity Management System credentials. The administrator can create and configure users and groups, assign roles and projects, and track service tickets.  They can also customize the ticketing system. 
+
+### JIRA Service Management 
+
+The administrator begins at Jira Service Management to create users, projects, define the organization, and configure the service ticket system.
+
+Go to Customers to add organizations and projects. Once added, click on the name of the organization to get the ID, needed later, from the URL. \
+ e.g. example.com/jira/servicedesk/projects/ECS/organization/3 → the organization ID is “3”
+
+### Entando Identity Management System 
+Logging into the Entando Identity Management System, you will see the landing page shown below. Like the App Builder, the left navigation bar is your guide for managing users, groups, and most importantly, roles. Using the RBAC model, define what access users have by the roles and groups they are assigned. Some important guidelines are noted below. 
+
+![Entando Identity Management System](./images/custportal-idmanagement-main.png)
+
+* **Define the Realm Setting**. \
+The `Realm` is a set of users, credentials, roles, and groups. A user belongs to and logs into a `Realm`. 
+
+* **Create Roles**\
+You can use the default roles by clicking on `Client Roles` and choosing `entandodemo-customerportal-server`. Access for each role is defined as follows:
+    * `cp-customer` -  assigned directly to specific projects for a single customer
+    * `cp-partner` - assigned directly to specific projects for multiple customers
+    * `cp-support` -  read only view of all customer projects
+    * `cp-admin` - admin access for the Customer Portal
+
+* **Create New Users**: \
+Each new user must have a Jira Service Management account before they can be added to the Entando ID Management System for access to the Customer Portal.
+     1. From the navigation bar, go to Users. Click `Add User` at right.
+     2. Complete the form as needed but note the requirements for the following fields:\
+       `Username`: a unique name\
+       `Email`: must use the same address used in Jira\
+       `User Enabled` → On\
+       `Email Verified` → Off\
+     `Save`
+
+     3. Send an email to the user to activate their account and set a new password: \
+     Go to `Credentials` header. \
+     Under `Credential Reset`, in the `Reset Actions` → `Update Password`. For external users, the recommendation is to extend the `Expires In` setting to a minimum of 2 days. \
```

Comment:
Let's remove the "For external users" sentence. That's guidance for our own use, not a technical note.